gax-java/gax-httpjson/src/main/java/com/google/api/gax/httpjson/HttpJsonInterceptorProvider.java

```diff
@@ -35,6 +35,7 @@
 /** Provider of custom REST ClientInterceptors. */
 @BetaApi(
     "The surface for adding custom interceptors is not stable yet and may change in the future.")
+@FunctionalInterface
```

Comment:
Can you remove this as well since it's beta?

Reply:
done